script.rb

```diff
@@ -0,0 +1,102 @@
+module Enumerable
+  # my_each
+  def my_each
+    i = 0
+    while i < to_a.length
+      yield to_a[i]
+      i += 1
+    end
+    self
+  end
```

Comment:
- [ ] enumerables #my_each returns an Enumerator if no block is given (return to_enum(:my_each) unless block_given?)